rules/chatgpt-cheat-sheet/rule.md

```diff
@@ -0,0 +1,20 @@
+---
+type: rule
+title: Do you have a ChatGPT cheat sheet?
+uri: chatgpt-cheat-sheet
+authors:
+  - title: Adam Cogan
+    url: https://www.ssw.com.au/people/adam-cogan
+created: 2023-04-14T19:11:11.002Z
+guid: c50ed7aa-1539-4ff9-a11c-d95cc0616d4e
+---
+As ChatGPT becomes increasingly popular, users are discovering the most effective ways to prompt the AI for different scenarios. One way to compile this information is by putting it in easy to read graphics called cheat sheets. Cheat sheets can spark ideas that you would not have otherwise thought of!  
+  
+            
+<!--endintro-->
+
+![ChatGPT-Cheat-sheet-A3](https://user-images.githubusercontent.com/130118701/232672071-31e21091-17ee-4276-9119-41c1a26f2ec0.jpg) **Figure: SSW ChatGPT Cheat Sheet**  
+
+  
+Related example: Did you know you can use ChatGPT to correct somebody's mistake in a courteous and fun way?  
```

Comment:
This is odd. Should this question be a link?
If there is no link... I would convert it to NOT be a question

Reply:
The link for this question is in the next line "Is everyone in your team a standards watchdog?"
I think it makes sense to ask the question and then link the rule directly afterwards. This can be either in the body section or in the related section.